build.cake

```diff
@@ -12,7 +12,7 @@ var configuration = Argument("configuration", "Release");
 //////////////////////////////////////////////////////////////////////
 
 var version = "3.11.0";
-var modifier = "";
+var modifier = "-d03";
```

Comment:
Is there going to be a prerelease publish?

Reply:
Just slipped through, I update these locally when debugging.  Will remove again.